src/utils/utils.js

```diff
@@ -40,10 +40,14 @@ export function getInitiativeDeckDiscardPile(strict = false) {
  * and shuffles them back into the initiative deck.
  * @param {boolean} [chatNotification=false] Whether to send a chat notification.
```

Comment:
Improving the documentation:
```suggestion
 * @param {boolean} [chatNotification=false] Whether to send a chat notification.
 * @param {number[]} [toExclude=[]] Array of card values to exclude from the reset.
```